config/locales/en.yml

```diff
@@ -415,8 +415,10 @@ en:
         title: "Matomo Settings"
         matomo_url: "Matomo URL"
         matomo_site_id: "Matomo Site ID"
-        info_html: "Matomo is a Web and Mobile Analytics. You can either host Matomo on-premises or use a cloud-hosted service. See <a href='http://matomo.org' target='_blank'>matomo.org</a> for more information."
+        matomo_tag_manager_url: "Matomo Tag Manager URL"
+        info_html: "Matomo is a Web and Mobile Analytics application. You can either host Matomo on-premises or use a cloud-hosted service. See <a href='http://matomo.org' target='_blank'>matomo.org</a> for more information."
         config_instructions_html: "Here you can configure the OFN Matomo integration. The Matomo URL below should point to the Matomo instance where the user tracking information will be sent to; if it is left empty, Matomo user tracking will be disabled. The Site ID field is not mandatory but useful if you are tracking more than one website on a single Matomo instance; it can be found on the Matomo instance console."
+        config_instructions_tag_manager_html: "Setting the Matomo Tag Manager url enables Matomo Tag Manager. This tool lets non developers set up analytics events. The Matomo Tag Manager URL is copied from the Install Code section of the Matomo Tag Manager. Ensure you select the right container and environment as these options change the URL."
```

Comment:
URL should be uppercase
"the Matomo Tag Manager URL enables"

I dont see the need to indicate who can do it, it could simply be:
"This tool lets you set up analytics events."

Reply:
Ok